### PR TITLE
feat: add information on monitoring a node

### PR DESCRIPTION
**Pull request by talalashraf**

### CODE REVIEW

**Comment by milapsheth** [comment]
Check if a validator is still a chain maintainer
`axelard q nexus chain-maintainers [chain]`

File: pages/validator/operations/monitoring.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import Callout from 'nextra-theme-docs/callout'
+
+# Monitoring a Validator or Node
+
+There are a few different aspects to monitoring your validator. We need to monitor both the infrastructure and the axelard & supporting proceeses.
```
Comment (kalidax) [inline pages/validator/operations/monitoring.md:5]:
```suggestion
There are a few different aspects to monitoring your validator. We need to monitor both the infrastructure and the axelard & supporting processes.
```

**Review by kalidax — COMMENTED** [review]
Maybe specifying (in a separate PR) that we recommend to run the monitoring on a dedicated node and we have to check that the node/validator itself is up (because if they run the monitoring on the same node and the node crash the monitoring will be useless).
A simple ping can do the stuff.

File: pages/validator/operations/monitoring.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+* Bandwidth
+
+You can run the node and quickly establish baselines for some of these like disk i/o, bandwidth etc.
+
+
+## Validator and Supporting Processes
+
+### Prometheus Metrics
+
+Axelar runs using Tendermint consensus. Several metrics are exposed via the prometheus metrics endpoint by Tendermint. You can find details on the metrics available through tendermint [here](https://docs.tendermint.com/v0.34/tendermint-core/metrics.html). All these metrics are prefixed with `tendermint` e.g `consensus_height` will be `tendermint_consensus_height`.
```
Comment (milapsheth) [inline pages/validator/operations/monitoring.md:22]: Not sure if the prefix holds for the custom metrics we add.

File: pages/validator/operations/monitoring.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+```
+
+You can create a script that can check output of this command. For instance to ensure tofnd status is passed you can use something like:
+```sh
+$AXELARD_HOME/bin/axelard health-check --tofnd-host localhost --operator-addr {VALOPER_ADDR} | grep tofnd | awk -F: '{print $2}' | tr -d ' '
+```
+
+#### Chain Maintainers
+
+For all the chains that are active on the network, make sure that your validator is a chain maintainer. The query for this is:
```
Comment (milapsheth) [inline pages/validator/operations/monitoring.md:82]: Only applies to EVM chains. Also mention how to get chains
`axelard q evm chains`

File: pages/validator/operations/monitoring.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+```sh
+axelard q nexus chain-maintainers <chain-name>
```
Comment (milapsheth) [inline pages/validator/operations/monitoring.md:84]:
```suggestion
axelard q nexus chain-maintainers [chain]
```
Keeping format consistent

File: pages/validator/operations/monitoring.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+```
+
+To test if your validator is a chain-maintainer, retreive the axelarvaloper address and look for that in the output. For example, for ethereum, the command would look like:
```
Comment (milapsheth) [inline pages/validator/operations/monitoring.md:87]:
```suggestion
To test if your validator is a chain-maintainer, retrieve the `axelarvaloper` address and look for that in the output. For example, for ethereum, the command would look like:
```

**Review by milapsheth — APPROVED** [review]

File: pages/validator/operations/monitoring.md (hunk 2 above)
Comment (talalashraf) [reply pages/validator/operations/monitoring.md:22]: only talking about tendermint metrics mentioned in the docs. not ours.

**Pull request merged**